regression/cbmc/memcpy4/main.c

```diff
@@ -0,0 +1,16 @@
+#include <assert.h>
+#include <string.h>
+
+int main()
+{
+  char foo[1];
+  char result = 0;
+  int size;
+  __CPROVER_assume(size == 1);
```

Comment:
Worth commenting that this weird construct is here to defeat constant propagation (I presume?) and shouldn't be optimised away.